### PR TITLE
Updates storage related documentations

### PR DESCRIPTION
**Pull request by KSanjith**
This PR resolves issue #218 , #163, #162  
- Updates the load recipe diagram to match the updated loading code 
- Updates the DG description of the storage to match the updated loading code
- Describes in detail how the save-load system works in the UG
- Describes in detail how the Recipes and Ingredients are different in the UG
- Renames save files to "cookbook.txt" and "inventory.txt"
- Updates the UG and Help Command UI to be more consistent and match updated features

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -44,13 +45,34 @@ It can also keep track of what ingredients you have on hand, and recommend recip
 
 ## Features
 
-### Auto-Save Databases
+### Recipes and Ingredients
+* There are 2 categories of data that YMFC will help you store: Recipes and Ingredients
+* The **Recipes** are stored in the **cookbook** database while **Ingredients** are stored in the **inventory** database
```
Comment (Sukkaito) [inline docs/UserGuide.md:50]: Here you mentioned about "cookbook" and "inventory". Should we rename our savefile that way for easier referencing?

File: docs/img/SequenceDiagrams/LoadRecipesSequenceDiagram.png
Comment (seanngja) [inline docs/img/SequenceDiagrams/LoadRecipesSequenceDiagram.png:1]: Nice, you were able to separate the delete symbol `X` from the activation bar. Now we just need to remember to remove the lifelines beyond the delete symbol before final submission T.T

**Review by seanngja — APPROVED** [review]
The diagrams look good to me. Could consider the points suggested by @Sukkaito and @3CCLY before merging.

File: .gitignore [hunk 2]
```diff
@@ -17,4 +17,7 @@ bin/
 text-ui-test/EXPECTED-UNIX.TXT
 
 /data/recipes.txt
-/data/ingredients.txt
\ No newline at end of file
+/data/ingredients.txt
+
+/data/cookbook.txt
+/data/inventory.txt
```
Comment (3CCLY) [inline .gitignore:23]: Good call on updating this too

**Review by 3CCLY — APPROVED** [review]
LGTM

**Pull request merged**